src/integrations/watchers/indexable-permalink-watcher.php

```diff
@@ -156,12 +178,21 @@ public function force_reset_permalinks() {
 	}
 
 	/**
-	 * Checks whether permalinks should be reset.
+	 * Checks what the reset reason is, and returns null otherwise.
 	 *
-	 * @return bool Whether the permalinks should be reset.
+	 * @return string The indexing reason.
```

Comment:
It can also return null so this notation should be changed.

Reply:
This function has been removed